specification/servicebus/resource-manager/Microsoft.ServiceBus/2017-04-01/servicebus.json

```diff
@@ -3103,6 +3223,237 @@
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of EventHubs."
+        }
+      },
+      "description": "The result of the List EventHubs operation."
+    },
+    "CaptureDescription": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "A value that indicates whether capture description is enabled. "
+        },
+        "encoding": {
+          "type": "string",
+          "description": "Enumerates the possible values for the encoding format of capture description.",
+          "enum": [
+            "Avro",
+            "AvroDeflate"
+          ],
+          "x-ms-enum": {
+            "name": "EncodingCaptureDescription",
+            "modelAsString": false
+          }
+        },
+        "intervalInSeconds": {
+          "format": "int32",
+          "type": "integer",
+          "maximum": 900,
+          "minimum": 60,
+          "description": "The time window allows you to set the frequency with which the capture to Azure Blobs will happen, value should between 60 to 900 seconds"
+        },
+        "sizeLimitInBytes": {
+          "format": "int32",
+          "type": "integer",
+          "maximum": 524288000,
+          "minimum": 10485760,
+          "description": "The size window defines the amount of data built up in your Event Hub before an capture operation, value should be between 10485760 to 524288000 bytes"
+        },
+        "destination": {
+          "$ref": "#/definitions/Destination",
+          "description": "Properties of Destination where capture will be stored. (Storage Account, Blob Names)"
+        }
+      },
+      "description": "Properties to configure capture description for eventhub"
+    },
+    "Destination": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name for capture destination"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "properties": {
+            "storageAccountResourceId": {
+              "type": "string",
+              "description": "Resource id of the storage account to be used to create the blobs"
+            },
+            "blobContainer": {
+              "type": "string",
+              "description": "Blob container Name"
+            },
+            "archiveNameFormat": {
+              "type": "string",
+              "description": "Blob naming convention for archive, e.g. {Namespace}/{EventHub}/{PartitionId}/{Year}/{Month}/{Day}/{Hour}/{Minute}/{Second}. Here all the parameters (Namespace,EventHub .. etc) are mandatory irrespective of order"
+            }
+          },
+          "description": "Properties describing the storage account, blob container and acrchive anme format for capture destination"
+        }
+      },
+      "description": "Capture storage details for capture description"
+    },
+    "RegionResource": {
+      "properties": {
```

Comment:
Not sure what you are aiming for, but is ```RegionResource``` an arm resource here? If yes, this needs appropriate changes.